pennylane/gradients/finite_difference.py

```diff
@@ -157,21 +156,15 @@ def finite_diff_coeffs(n, approx_order, strategy):
     return coeffs_and_shifts
 
 
-def _processing_fn(results, shots=None, single_shot_batch_fn=None):
-    shot_vector = isinstance(shots, Sequence)
-
-    if not shot_vector:
-        grads_tuple = single_shot_batch_fn(results)
-    else:
-        grads_tuple = []
-        len_shot_vec = _get_num_copies(shots)
-        for idx in range(len_shot_vec):
-            res = [tape_res[idx] for tape_res in results]
-            g_tuple = single_shot_batch_fn(res)
-            grads_tuple.append(g_tuple)
-        grads_tuple = tuple(grads_tuple)
-
-    return grads_tuple
+def _processing_fn(results, shots=qml.measurements.Shots(None), single_shot_batch_fn=None):

```

Comment:
```suggestion
def _processing_fn(results, shots: "Shots"=qml.measurements.Shots(None), single_shot_batch_fn=None):

```
Might be helpful to add a type hint so its clear that the keyword argument has to be a shots object.